momento-sdk/src/main/java/momento/sdk/PreviewStorageClientBuilder.java

```diff
@@ -49,7 +49,7 @@ public PreviewStorageClientBuilder withConfiguration(
    *
    * @return the client.
    */
-  public IPreviewStorageClient build() {
+  public PreviewStorageClient build() {
```

Comment:
I changed this since `PreviewStorageClient` is `Closeable`. In order to both use the builder and explicitly call `close` on the client, we should return the concrete class.